app/core/edge_inference.py

```diff
@@ -1,19 +1,19 @@
+import io
 import logging
 import os
 import shutil
 import socket
 import time
 from typing import Dict, Optional
 
-import numpy as np
 import requests
-import tritonclient.http as tritonclient
```

Comment:
This file is where most of the changes are - swapping out triton's client and some changes to our model repository structure.

For now I just set us up to use `requests` to connect to the inference server, but an auto-generated client would be a good next step.